### DISCUSSION TITLE
1.33.1

### DISCUSSION THREAD

**Discussion by Koenkk** [Koenkk/zigbee2mqtt#19154] [General]
# Improvements
- [#6201](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6201) Expose `low_water_temp_protection` for Sinopé RM3500ZB (@alray31)
- [#6202](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6202) Support color for Philips 7602031N6 (@meeb)
- [#6190](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6190) Support min/max brightness and countdown for TuYa TS0601_dimmer_3 (@kuzaxak)
- [#6180](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6180) Add OTA support for Xiaomi `LLKZMK12LM` (@Otnow)
- [#6172](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6172) Expose `power_on_behavior` and `backlight_mode` for TuYa TS0601_dimmer_1, TS0601_dimmer_2 and TS0601_dimmer_3 (@kirovilya)
- [#6163](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6163) Expose color options for Ikea lights (@protyposis)
- [#6159](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6159) Expose power outage memory for Mercator Ikuü SPP02GIP (@georg)
- [#6145](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6145) Add battery for W599501 (@Koenkk)
- [#6146](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6146) Support battery for TRVZB (@liangjia2019)
- [#18682](https://github.com/Koenkk/zigbee2mqtt/issues/18682) Detect all TuYa TS0202 as supported (@Koenkk)
- [#18850](https://github.com/Koenkk/zigbee2mqtt/issues/18850) Support color for HORNBACH 10454471 (@Koenkk)
- [#18745](https://github.com/Koenkk/zigbee2mqtt/issues/18745) Support more features for TuYa TS030F  (@JuMi2006)
- [#18818](https://github.com/Koenkk/zigbee2mqtt/issues/18818) Detect all TuYa TS0014 as supported (@Koenkk)
- [#763](https://github.com/Koenkk/zigbee-herdsman/issues/763) Support mdns discovery for all adapter instead of just zstack (@Tarik2142)

# Fixes
- [#6213](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6213) Correct typos (@Rotzbua)
- [#6212](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6212) Fix typo (@Rotzbua)
- [#18791](https://github.com/Koenkk/zigbee2mqtt/issues/18791) Fix `_TZE200_5toc8efa` `current_heating_setpoint` multiplied by 10 (@Koenkk)
- [#18413](https://github.com/Koenkk/zigbee2mqtt/issues/18413) Fix battery missing for Zemismart ZM85EL-2Z (@Koenkk)
- [#6217](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6217) Fix invalid device model for `TRADFRI bulb GU10 WS 345lm` (@bjorne)
- [#19091](https://github.com/Koenkk/zigbee2mqtt/issues/19091) Rename `BHT-006GBZB` to `BHT-002/BHT-006` (@Koenkk)
- [#6193](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6193) Fix Legrand OTA not working (@FabianMangold)
- [#18939](https://github.com/Koenkk/zigbee2mqtt/issues/18939) Fix battery missing for `_TZE200_pw7mji0l` (Zemismart ZM25EL) (@Koenkk)
- [#19036](https://github.com/Koenkk/zigbee2mqtt/issues/19036) Fix power source missing for 1402769 (@Koenkk)
- [#19011](https://github.com/Koenkk/zigbee2mqtt/issues/19011) Improve missing TuYa datapoint logging (@Koenkk)
- [#6187](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6187) Fix inconsistent second units (s) (@Koenkk)
- [#6182](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6182) Arm mode and melody fix for NAS-AB06B2 (@Burki24)
- [#6181](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6181) Fix PJ-1203A power_factor divided by 100 (@alexandrebelloni)
- [#6145](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6145) Fix power source for Schneider Electric W599001/W599501 (@Koenkk)
- [#18987](https://github.com/Koenkk/zigbee2mqtt/discussions/18987) Rename LED2103GS to LED2103G5 (@Koenkk)
- [#6175](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6175) Restore power_on_behavior for Gledopto GL-C-006P and GL-LB-001P (@ov1d1u)
- [#18961](https://github.com/Koenkk/zigbee2mqtt/issues/18961) Fix Innr BY 266 and RS 227 T not controllable (@Koenkk)
- [#19001](https://github.com/Koenkk/zigbee2mqtt/issues/19001) Fix ZNLDP13LM power source and device type (@Koenkk)
- [#6154](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6154) Fix unit missing for `voltage_phase_c` (@esplor)
- [#6141](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6141) Rename 948.47 to 948.47/29165 and add color temp range (@unbekannt3)
- [#6135](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6135) Fix 9290030517 discovery (@zegl)
- [#18846](https://github.com/Koenkk/zigbee2mqtt/issues/18846) Remove unsupported color from Gledopto GL-B-002P and GL-B-004P (@Koenkk)
- [#6131](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6131) Fix TuYa YXZBRB58 `detection_delay` and `fading_time` (@blakadder)
- [#18832](https://github.com/Koenkk/zigbee2mqtt/issues/18832) Add back tamper for TuYa TS0202 (@Koenkk)
- [#18821](https://github.com/Koenkk/zigbee2mqtt/issues/18821) Rename `LED1923R5` to `LED1923R5/LED1925G6` (@Koenkk)
- [#17008](https://github.com/Koenkk/zigbee2mqtt/issues/17008) Expose `commandMcuVersionResponse` (@Koenkk)
- [#756](https://github.com/Koenkk/zigbee-herdsman/issues/756) Fix incorrect configured reporting cluster returned when cluster is manufacturer specific (@Koenkk)

# New supported devices
This release adds support for 41 devices: 
- [#6216](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6216) `552-720X1` Niko Battery switch with 1 button (@FaerimWoodsong)
- [#6216](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6216) `552-720X2` Niko Battery switch with 2 buttons (@FaerimWoodsong)
- [#6218](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6218) `BSEN-CV` Bosch Door/window contact II (@smarts4home)
- [#19118](https://github.com/Koenkk/zigbee2mqtt/issues/19118) `LED2201G8` IKEA TRADFRI LED bulb E27 1055 lumen, dimmable, white spectrum (@Koenkk)
- [#5788](https://github.com/Koenkk/zigbee-herdsman-converters/issues/5788) `MOT-C1ZxxC/F` Profalux Cover (@sven337)
- [#6204](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6204) `4512765` Namron Zigbee humidity and temperature Sensor (@joinn1710)
- [#6200](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6200) `9290035753` Philips Hue White and Color Ambiance GU5.3 (@mundschenk-at)
- [#18704](https://github.com/Koenkk/zigbee2mqtt/issues/18704) `BLE-YL01` TuYa Smart WiFi Zigbee chlorine meter (@cloudbr34k84)
- [#19073](https://github.com/Koenkk/zigbee2mqtt/issues/19073) `8719514491229` Philips Hue White and Color Ambiance E14 (@ckref)
- [#19054](https://github.com/Koenkk/zigbee2mqtt/issues/19054) `9290035639` Philips Hue Secure contact sensor (@nullcreek)
- [#19064](https://github.com/Koenkk/zigbee2mqtt/issues/19064) `929003596101` Philips Hue Filament Edison bulb ST64 550lm E26 (@sympapa)
- [#6194](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6194) `MEG5116-0300/MEG5171-0000` Schneider Electric Merten MEG5171 PlusLink Dimmer insert with Merten Wiser System M Push Button (1fold) (@mhenreaux)
- [#18963](https://github.com/Koenkk/zigbee2mqtt/issues/18963) `rtsc11r` TuYa 5.8G human presence sensor with relay (@MasterFeige)
- [#6174](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6174) `TYBAC-006` TuYa Wall-mount thermostat for 2-pipe fan-coil unit (@prsveloso)
- [#19020](https://github.com/Koenkk/zigbee2mqtt/issues/19020) `8718696126523` Philips Hue Phoenix ceiling light (@Koenkk)
- [#6178](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6178) `8719514491342` Philips Hue white ambiance MR16 with Bluetooth (@Balsvik)
- [#6179](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6179) `9290036745` Philips Hue Festavia gradient light string 100 (@mundschenk-at)
- [#6173](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6173) `SS300` Cleverio Temperature/humdity sensor (@nightcbis)
- [#12883](https://github.com/Koenkk/zigbee2mqtt/issues/12883) `TS0601_pir` TuYa Haozee PIR sensor (@Bacchus777)
- [#6176](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6176) `VM-Zigbee-S02-0-10V` Lonsonho 2 channel Zigbee 0-10V dimmer module (@LiZhenhuan1019)
- [#19018](https://github.com/Koenkk/zigbee2mqtt/issues/19018) `ZK03839` Viessmann ViCare climate sensor (@Koenkk)
- [#6169](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6169) `W45CZ` ORVIBO Smart curtain motor (@EarthEclipse)
- [#6164](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6164) `NAS-AB06B2` Neo Outdoor solar alarm (@Burki24)
- [#18856](https://github.com/Koenkk/zigbee2mqtt/issues/18856) `LLKZMK12LM` Xiaomi Aqara dual relay module T2 (@kei81131)
- [#18371](https://github.com/Koenkk/zigbee2mqtt/issues/18371) `TS0601_switch_12` TuYa ZXYH 12 gang switch (@Franckybel)
- [#6156](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6156) `RB56SC` Acuity Brands Lighting (ABL) Juno Retrobasics 4" and 6" LED smart downlight (@peetipablo)
- [#6155](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6155) `ZWT198` TuYa ONNDO smart thermostat (@fume)
- [#18919](https://github.com/Koenkk/zigbee2mqtt/issues/18919) `9290030516` Philips Hue filament standard A60/E27 Bluetooth (@tanders587)
- [#18103](https://github.com/Koenkk/zigbee2mqtt/issues/18103) `TS0601_cover_7` TuYa Cover motor (@Koenkk)
- [#18915](https://github.com/Koenkk/zigbee2mqtt/issues/18915) `929003596001` Philips Hue filament A60 550lm E26 (@sympapa)
- [#6149](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6149) `TS0601_illuminance_temperature_humidity_sensor_2` TuYa Illuminance sensor (@mrakister)
- [#6147](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6147) `LED2104R3` IKEA TRADFRI LED bulb GU10 WW 345 lumen, dimmable (@liquidacid)
- [#6143](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6143) `10454467` HORNBACH FLAIR Viyu smart LED candle RGB E14 (@modanOU)
- [#18884](https://github.com/Koenkk/zigbee2mqtt/issues/18884) `1822647A` SOMFY Zigbee smart plug (@Koenkk)
- [#18870](https://github.com/Koenkk/zigbee2mqtt/issues/18870) `GL-LB-001P` Gledopto Zigbee USB LED bar RGB+CCT (pro) (@Koenkk)
- [#6144](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6144) `SNZB-06P` SONOFF Zigbee occupancy sensor (@liangjia2019)
- [#6144](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6144) `TRVZB` SONOFF Zigbee thermostat (@liangjia2019)
- [#6138](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6138) `ROB_200-063-0` ROBB Zigbee 0-10V PWM dimmer (@TimEwaltsIT)
- [#6130](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6130) `929003536001` Philips Hue Iris copper special edition (generation 4) (@davis-griffin)
- [#18844](https://github.com/Koenkk/zigbee2mqtt/issues/18844) `SLT3d` Hive Heating thermostat remote control (@Koenkk)
- [#18996](https://github.com/Koenkk/zigbee2mqtt/issues/18996) `LED2103G5` IKEA TRADFRI LED bulb E27 806 lumen, wireless dimmable warm white (@yaskad)

# Fixed device detections
- [#6145](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6145) Detect `W599501` as Schneider Electric W599001 (@ronniebach)
- [#6208](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6208) Detect `_TZ3000_h8ngtlxy` as TuYa ZN231392 (@garbagecollected)
- [#19105](https://github.com/Koenkk/zigbee2mqtt/issues/19105) Detect `_TZ3000_lf56vpxj` as TuYa ZP01 (@Koenkk)
- [#6210](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6210) Detect `_TZ3000_zl1kmjqx` as TuYa IH-K009 (@fsaris)
- [#19130](https://github.com/Koenkk/zigbee2mqtt/issues/19130) Detect `_TZ3210_rcggc0ys` as Moes ZLD-RCW_2 (@Koenkk)
- [#19117](https://github.com/Koenkk/zigbee2mqtt/issues/19117) Detect `_TZE200_bkkmqmyo` as Hiking DDS238-2 (@Koenkk)
- [#19103](https://github.com/Koenkk/zigbee2mqtt/issues/19103) Detect `_TZE200_rtrmfadk` as TuYa TS0601_thermostat_1 (@Koenkk)
- [#19101](https://github.com/Koenkk/zigbee2mqtt/issues/19101) Detect `3450-Geu` as Iris 3450-L (@Koenkk)
- [#19086](https://github.com/Koenkk/zigbee2mqtt/issues/19086) Detect `_TZ3000_qeuvnohg` as Tongou TO-Q-SY1-JZT (@Koenkk)
- [#19080](https://github.com/Koenkk/zigbee2mqtt/issues/19080) Detect `_TZE204_rhblgy0z` as TuYa TS0601_din_3 (@Koenkk)
- [#6198](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6198) Detect `_TZE204_upagmta9` as TuYa ZTH05 (@francisbesset)
- [#19084](https://github.com/Koenkk/zigbee2mqtt/discussions/19084) Detect `TRADFRIbulbG125E26WSopal470lm` as IKEA LED1936G5 (@Koenkk)
- [#18337](https://github.com/Koenkk/zigbee2mqtt/issues/18337) Detect `_TZE204_ptaqh9tk` as TuYa TS0601_switch (@Koenkk)
- [#6196](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6196) Detect `SMARTCODE_DEADBOLT_10_W3_L` as Kwikset 99140-031L (@onixx)
- [#6185](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6185) Detect `E220-KR5N0Z0-HA` as LELLKI WP33-EU/WP34-EU (@gdjang)
- [#19024](https://github.com/Koenkk/zigbee2mqtt/discussions/19024) Detect `SWITCH-ZR03-2` as eWeLink ZB-SW02  (@zdenekstepanek)
- [#6170](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6170) Detect `_TZE200_1fuxihti` as TuYa TS0601_cover_1 (@Koenkk)
- [#18734](https://github.com/Koenkk/zigbee2mqtt/issues/18734) Detect `_TZE200_rks0sgb7` as TuYa PJ-1203A  (@1vanj0)
- [#6160](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6160) Detect `_TZ3000_skueekg3` as TuYa WHD02 (@alray31)
- [#6162](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6162) Detect `_TZ3000_uwkja6z1` as Nous A4Z (@Koenkk)
- [#18954](https://github.com/Koenkk/zigbee2mqtt/issues/18954) Detect `_TZE200_gaj531w3` as Yushun YS-MT750L (@Koenkk)
- [#6136](https://github.com/Koenkk/zigbee-herdsman-converters/pull/6136) Detect `_TZE204_hlx9tnzb` as ZS-SR-EUD-1 and `_TZE204_1v1dxkck` as `ZS-SR-EUD-3` (@Koenkk)
- [#18336](https://github.com/Koenkk/zigbee2mqtt/issues/18336) Detect `_TZE204_wvovwe9h` as TuYa TS0601_switch_2_gang  (@BandBxx)
- [#18942](https://github.com/Koenkk/zigbee2mqtt/issues/18942) Detect `929003597801` as 3216131P6  (@MistaWu)
- [#18847](https://github.com/Koenkk/zigbee2mqtt/issues/18847) Detect `_TZE200_a0syesf5` as Mercator Ikuü SSWRM-ZB (@Koenkk)
- [#6134](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6134) Detect `Wireless Scenes Command` as Legrand 067755 (@jiquem)
- [#18682](https://github.com/Koenkk/zigbee2mqtt/issues/18682) Detect `_TZ3000_jmrgyl7o` as Luminea ZX-5311 (@Koenkk)
- [#18745](https://github.com/Koenkk/zigbee2mqtt/issues/18745) Detect `_TZB000_42ha4rsc` as Lidl HG09648 (@Koenkk)
- [#17008](https://github.com/Koenkk/zigbee2mqtt/issues/17008) Detect `_TZE200_cirvgep4z` as TuYa ZTH08-E (@Koenkk)
- [#6136](https://github.com/Koenkk/zigbee-herdsman-converters/issues/6136) Detect `_TZE204_zenj4lxv` as Moes ZS-SR-EUD-2 (@kirovilya)
- [#18799](https://github.com/Koenkk/zigbee2mqtt/issues/18799) Detect `Dimmable`/`Paulmann Licht GmbH` as 93999 (@Koenkk)

<hr /><em>This discussion was created from the release <a href='https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.33.1'>1.33.1</a>.</em>

**Comment by RusselIce** [comment]
Good evening. After the update, the _TZE204_6fk3gew device went into unsupported status. The new pairing did not change anything. and everything disappeared from the Properties tab

**Reply by Koenkk** [reply]
I checked, but as far as I can see this device was not supported in 1.33.0, please fill in https://github.com/Koenkk/zigbee2mqtt/issues/new?assignees=&labels=new+device+support&projects=&template=new_device_support.yaml&title=%5BNew+device+support%5D%3A+

**Comment by prsveloso** [comment]
Hi.
After testing this update I can see that _TZE204_mpbki2zm is showing the wrong day of the week.
For today (Tuesday) it shows Sunday.
If I use, in my converter, onEvent: tuya.onEventSetLocalTime, it works just fine.
Also, manual_mode is not working: No converter available for 'manual_mode' ("ON")
Thanks.

**Comment by gelimelech** [comment]
Is it possible to add support to Heiman HS1SA-E (smart smoke sensor)?
currently its not supported

https://www.aliexpress.com/item/4000136474999.html?spm=a2g0o.order_list.order_list_main.5.21ef1802mayOdM

**Reply by alexandrebelloni** [reply]
It is supported:
https://www.zigbee2mqtt.io/devices/HS1SA.html

**Reply by gelimelech** [reply]
thanks! 
it wasn't supported few weeks back.

**Reply by alexandrebelloni** [reply]
weird, it has been about a year and a half that I have five working. Pairing them is a pain though as you have to keep pressing the button regularly which makes it sound the test alarm.

**Comment by JustAnotherPU** [comment]
any idea if a fix is in the pipeline for aqara mini bouton ? still can't pair it at all

**Comment by joey0422** [comment]
Hello
After the auto update to 1.33.1, all the zigbee devices stopped working three days ago. Showing 502 bad gateway after manual start the z2m. Tried to restart the HA and does not work. Please give a hint to solve this situation. Thank you.

Log
Zigbee2MQTT
[09:12:05] INFO: Preparing to start...
[09:12:06] INFO: Socat not enabled
[09:12:07] INFO: Starting Zigbee2MQTT...
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
            READ THIS CAREFULLY
Refusing to start because configuration is not valid, found the following errors:
- MQTT retention requires protocol version 5
If you don't know how to solve this, read https://www.zigbee2mqtt.io/guide/configuration
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

**Reply by alray31** [reply]
What version were you on before updating? What broker are you using?

**Reply by joey0422** [reply]
It was Z2M 1.33.0 with Mosquitto broker 6.3.1.

Now the problem has been solved.
Tried restore to the 1.33.0 backup then add version: 5 in the file editor
yaml NOT configuration in z2m.
Then reset mosquitto broker login account and password.
Finally update again to z2m 1.33.7.
Restart the HA. Done.
Z2m is accessible via webUI.

**Comment by tellok** [comment]
Has anyone found out how to set Interlock mode with LLKZMK12LM?
